vagrant/base.sh

```diff
@@ -17,23 +17,33 @@
 
```

Comment:
I noticed the shebang line is in the wrong place, both here and in the other scripts in `vagrant/`. Wouldn't hurt to clean that up in this patch even though it wasn't introduced here. It looks like we're currently relying on `bash` being the default shell in the VMs because of this error.